src/main/java/training/subsystems/ArmSubsystems/elbowSubsystem/ElbowConstants.java

```diff
@@ -0,0 +1,66 @@
+package training.subsystems.ArmSubsystems.elbowSubsystem;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ElbowConstants {
+
+    public static final int ELBOW_ID = 5;
```

Comment:
some things:
create neo constants, simulation constants for constants that is type dependent
you don't need ELBOW before each thing here because u are in class named **Elbow**Constants